Unit test failed for DruidProcessingConfigTest.testDeserialization

Hi,

Unit test failed for DruidProcessingConfigTest.testDeserialization. Did any one faced this issue?

Stack Trace:
sts: 
  DruidProcessingConfigTest.testDeserialization:45 null

Tests run: 9551, Failures: 1, Errors: 0, Skipped: 13

[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] io.druid:druid .................................... SUCCESS [12.957s]
[INFO] druid-api ......................................... SUCCESS [3:15.808s]
[INFO] druid-common ...................................... SUCCESS [1:52.465s]
[INFO] druid-processing .................................. FAILURE [27:07.759s]
[INFO] druid-aws-common .................................. SKIPPED
[INFO] druid-server ...................................... SKIPPED
[INFO] druid-examples .................................... SKIPPED
[INFO] druid-indexing-hadoop ............................. SKIPPED
[INFO] druid-indexing-service ............................ SKIPPED
[INFO] druid-services .................................... SKIPPED
[INFO] druid-s3-extensions ............................... SKIPPED
[INFO] druid-datasketches ................................ SKIPPED
[INFO] druid-kafka-eight ................................. SKIPPED
[INFO] druid-histogram ................................... SKIPPED
[INFO] mysql-metadata-storage ............................ SKIPPED
[INFO] druid-integration-tests ........................... SKIPPED
[INFO] druid-benchmarks .................................. SKIPPED
[INFO] druid-avro-extensions ............................. SKIPPED
[INFO] druid-hdfs-storage ................................ SKIPPED
[INFO] druid-namespace-lookup ............................ SKIPPED
[INFO] druid-kafka-extraction-namespace .................. SKIPPED
[INFO] druid-kafka-indexing-service ...................... SKIPPED
[INFO] postgresql-metadata-storage ....................... SKIPPED
[INFO] druid-azure-extensions ............................ SKIPPED
[INFO] druid-cassandra-storage ........................... SKIPPED
[INFO] druid-rocketmq .................................... SKIPPED
[INFO] druid-cloudfiles-extensions ....................... SKIPPED
[INFO] graphite-emitter .................................. SKIPPED
[INFO] druid-kafka-eight-simple-consumer ................. SKIPPED
[INFO] druid-rabbitmq .................................... SKIPPED
[INFO] druid-distinctcount ............................... SKIPPED
[INFO] druid-parquet-extensions .......................... SKIPPED
[INFO] distribution ...................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 32:44.532s
[INFO] Finished at: Sat Apr 02 22:38:09 PDT 2016
[INFO] Final Memory: 28M/165M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.18.1:test (default-test) on project druid-processing: There are test failures.
[ERROR] 
[ERROR] Please refer to /home/cloudera/git/druid/processing/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :druid-processing

--- a/processing/src/test/java/io/druid/query/DruidProcessingConfigTest.java
+++ b/processing/src/test/java/io/druid/query/DruidProcessingConfigTest.java
@@ -42,7 +42,11 @@ public class DruidProcessingConfigTest
 
     Assert.assertEquals(1024 * 1024 * 1024, config.intermediateComputeSizeBytes());
     Assert.assertEquals(Integer.MAX_VALUE, config.poolCacheMaxCount());
-    Assert.assertTrue(config.getNumThreads() < Runtime.getRuntime().availableProcessors());
+    if (Runtime.getRuntime().availableProcessors() == 1) {
+      Assert.assertTrue(config.getNumThreads() == 1);
+    } else {
+      Assert.assertTrue(config.getNumThreads() < Runtime.getRuntime().availableProcessors());
+    }
     Assert.assertEquals(0, config.columnCacheSizeBytes());
     Assert.assertFalse(config.isFifo());
 

